sql-plugin/src/main/scala/com/nvidia/spark/rapids/python/PythonConfEntries.scala

```diff
@@ -0,0 +1,73 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.python
+
+import com.nvidia.spark.rapids.RapidsConf.conf
+
+object PythonConfEntries {
+
+  val PYTHON_GPU_ENABLED = conf("spark.rapids.python.gpu.enabled")
+    .doc("Enable (true) or disable (false) the support of running Python Pandas UDFs" +
+      " on the GPU. When enabled, Pandas UDFs can call cuDF APIs for acceleration." +
+      " This is an experimental feature.")
+    .booleanConf
+    .createWithDefault(false)
+
+  val CONCURRENT_PYTHON_WORKERS = conf("spark.rapids.python.concurrentPythonWorkers")
+    .doc("Set the number of Python worker processes that can execute concurrently per GPU. " +
+      "Python worker processes may temporarily block when the number of concurrent Python " +
+      "worker processes started by the same executor exceeds this amount. Allowing too " +
+      "many concurrent tasks on the same GPU may lead to GPU out of memory errors. " +
+      ">0 means enabled, while <=0 means unlimited")
+    .integerConf
+    .createWithDefault(0)
+
+  val PYTHON_RMM_ALLOC_FRACTION = conf("spark.rapids.python.memory.gpu.allocFraction")
+    .doc("The fraction of total GPU memory that should be initially allocated " +
+      "for pooled memory for all the Python workers. It supposes to be less than " +
+      "(1 - $(spark.rapids.memory.gpu.allocFraction)), since the executor will share the " +
+      "GPU with its owning Python workers.")
+    .doubleConf
+    .checkValue(v => v >= 0 && v <= 1, "The fraction value for Python workers must be in [0, 1].")
+    .createOptional
+
+  val PYTHON_RMM_MAX_ALLOC_FRACTION = conf("spark.rapids.python.memory.gpu.maxAllocFraction")
+    .doc("The fraction of total GPU memory that limits the maximum size of the RMM pool " +
+      "for all the Python workers. It supposes to be less than " +
+      "(1 - $(spark.rapids.memory.gpu.maxAllocFraction)), since the executor will share the " +
+      "GPU with its owning Python workers. when setting to 0 means no limit.")
+    .doubleConf
+    .checkValue(v => v >= 0 && v <= 1, "The value of maxAllocFraction for Python workers must be" +
+      " in [0, 1].")
+    .createWithDefault(0.0)
+
+  val PYTHON_POOLED_MEM = conf("spark.rapids.python.memory.gpu.pooling.enabled")
+    .doc("Should RMM in Python workers act as a pooling allocator for GPU memory, or" +
+      " should it just pass through to CUDA memory allocation directly.")
+    .booleanConf
+    .createOptional
+
+  val PYTHON_UVM_ENABLED = conf("spark.rapids.python.memory.uvm.enabled")
+    .doc("Similar with `spark.rapids.python.memory.uvm.enabled`, but this conf is for " +
+      "python workers. This is an experimental feature.")
+    .booleanConf
+    .createOptional
```

Comment:
We should mark this one as internal for now.  That was the decision for other UVM configs, at least until we can come up with a testing plan for them.

Reply:
done.